_i18n/en.yml

```diff
@@ -282,6 +282,18 @@ press-kit:
   promographicsp: Various graphics and Badges created by TheMonera, for both merchants and simple enthusiasts.
   othersymbols: Other symbols and logos
   mrllogo_para: The logo of the Monero Research Lab in SVG format.
+  contacts: Contacts
+  contactsp: There are multiple ways to contact the community. Depending by what information you are looking for, you might want to contact a workgroup, single members of the community, or the Core Team.
+  contactgen: "For press inquiries:"
+  contactwg: "To contact a specific Monero workgroup:"
+  contactcomm: "To contact the many subcommunities:"
+  whymonero: Why Monero?
+  whycommunity: Monero has one of the largest communities of any open-source project. Monero researches, audits, and implements critical security and privacy technologies, which are user by millions worldwide.
+  whycommunity1: Monero is trusted for more private transactions than all other cryptocurrencies combined.
+  whyconf: Monero is the only cryptocurrency project to be represented by its own village at Defcon, Grayhap and the CCC (Chaos Computer Club). Monero also hosts the Monero Konferenco, an academic conference, annually.
```

Comment:
Replace with the following please:

The Monero community has substantial involvement at Defcon, Grayhat and the CCC (Chaos Computer Club). The Monero Konferenco, an academic conference, is hosted annually.

Reply:
Changed